scripts/install_moodle.sh

```diff
@@ -181,7 +181,8 @@
     mkdir -p /moodle/html
     mkdir -p /moodle/certs
     mkdir -p /moodle/moodledata
-    chown -R www-data.www-data /moodle
+    mkdir -p /var/run/moodle/localcache
```

Comment:
Do we really need this directory on the controller machine? Is it just for the initial Moodle instllation?